templates/zerver/reset.html

```diff
@@ -32,6 +32,9 @@ <h1 class="get-started">{{ _('Reset your password') }}</h1>
                     </div>
                 </div>
             </form>
+            <div class="actions" if-zulip-electron><!-- only show if on `ZulipElectron` -->
```

Comment:
Is this valid HTML?  Why not just make it a class?  Would also prefer the name "desktop-app-only".

Reply:
Yep, attributes like that are valid HTML.

Reply:
Also think that "desktop-app-only" feels less prescise than "if-zulip-electron" for two reasons:

1. "Desktop App" could still ambiguously refer to the desktop web app, whereas electron wouldn't.
2. This more clearly and less ambiguously states that this is only supposed to appear *if* a condition is met.

Aside from that, I think an attribute is more apparent for people to see that this is a unique case, and wrt performance and support, this is equal to using class names.

Reply:
I guess it looks weird, and I think given how many contributors we have, it's best to not use obscure HTML features like this; a CSS class with the same name would work, so just use that.

I think calling it `electron-app-only` is reasonable.